fetch.bs

```diff
@@ -2127,12 +2145,25 @@ identified by a <b>key</b> (a <a>network partition key</a>), an <b>origin</b> (a
 <a for=/>origin</a>), and <b>credentials</b> (a boolean).
 
 <p>To <dfn export id=concept-connection-obtain>obtain a connection</dfn>, given a <var>key</var>,
-<var>origin</var>, and <var>credentials</var>, run these steps:
-
+<var>origin</var>, <var>credentials</var>, an optional boolean
+<dfn export for=obtain-a-connection><var>forHTTP3</var></dfn> (default: false), and an optional
```

Comment:
Also these probably shouldn't be definitions. They appear in bold now in the preview, unlike the other "given" variables.

Reply:
I think this is what @annevk wanted (see https://github.com/whatwg/fetch/pull/1171#discussion_r577534768).

Reply:
@jan-ivar see https://infra.spec.whatwg.org/#algorithm-params.